lib/testing-helpcenter/data/toolbar_info.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module TestingHelpCentreOnlyoffice
+  class ToolbarInfo
+    def self.administration_links
+      {
+        'Managing portals': 'navitem_manageportals',
+        'Reinforcing security': 'navitem_security',
+        'Configuring modules and tools': 'navitem_configuring',
+        'Customizing modules and tools': 'navitem_customizing',
+        'Managing users': 'navitem_manageusers',
+        'Connecting external services': 'navitem_externalservices',
+        'ONLYOFFICE Control Panel': 'navitem_controlpanel'
+      }
+    end
+
+    def self.integration_links
+      {
+        'Alfresco': 'navitem_integration_alfresco',
+        'Confluence': 'navitem_integration_confluence',
+        'HumHub': 'navitem_integration_hh',
+        'Liferay': 'navitem_integration_liferay',
+        'Nextcloud': 'navitem_integration_nextcloud',
+        'Nuxeo': 'navitem_integration_nuxeo',
+        'ownCloud': 'navitem_integration_ownclooud',
+        'Plone': 'navitem_integration_plone',
+        'SharePoint': 'navitem_integration_share'
+      }
+    end
+
+    def self.installation_links
+      {
+        'ONLYOFFICE Workspace': 'navitem_workspace',
+        'Workspace': 'navitem_opensource',
+        'Workspace Enterprise Edition': 'navitem_enterprise',
+        'Workspace Cloud Service': 'navitem_cloud',
+        'ONLYOFFICE Groups': 'navitem_community',
+        'ONLYOFFICE Mail': 'navitem_mail',
+        'ONLYOFFICE Talk': 'navitem_talk',
+        'ONLYOFFICE Docs': 'navitem_document',
+        'Community Edition': 'navitem_document_ce',
+        'Enterprise Edition': 'navitem_document_ee',
+        'Developer Edition': 'navitem_document_de',
+        'ONLYOFFICE Desktop Editors': 'navitem_desktop_install',
+        'ONLYOFFICE Mobile Apps': 'navitem_mobile_install'
+      }
+    end
+
+    def self.user_guides_links
+      {
+        'ONLYOFFICE Docs': 'F',
```

Comment:
Not a typo? Just `F`?

Reply:
Yes, just `F` 
<img width="747" alt="Screen Shot 2021-06-02 at 17 23 45" src="https://user-images.githubusercontent.com/40513035/120567741-4cd70400-c3c7-11eb-9f39-a8cfe12f20ca.png">